packages/stream_chat_flutter/lib/src/image_actions_modal.dart

```diff
@@ -103,11 +103,11 @@ class ImageActionsModal extends StatelessWidget {
                         color: StreamChatTheme.of(context).colorTheme.accentRed,
                       ),
                       () {
+                        var channel = StreamChannel.of(context).channel;
+                        channel.updateMessage(
+                            message..attachments.removeAt(currentIndex));
```

Comment:
This should not be in this pr, right?

Reply:
Think I branched from the wrong thing, fixed.